content/hi/docs/tutorials/update/_index.md

```diff
@@ -0,0 +1,4 @@
+---
+title: अपना ऐप अपडेट करें
+weight: 60
+---
```

Comment:
File location is wrong.

Reply:
can you specify?

Reply:
File should be `content/hi/docs/tutorials/kubernetes-basics/update/_index.md`
Also seems like `content/hi/docs/tasks/tools/included/kubectl-convert-overview.md` is mistakenly commited to this PR, can you please remove it. We prefer doing one file per PR. Thank you